### PR TITLE
[docs] Remove public google site verification link

### PR DESCRIPTION
**Pull request by black-dragon74**
The UID listed on this page is supposed to be kept private. Also,
this change just hides the hyperlink from the NAV, google crawlers
are still able to verify the site for authenticity. The way SSGs work,
if a file exists let's say "siteurl/myfile.html" and the route
"myfile.html" is not defined then the file at that location will be served
instead. That's why siteurl/googleXXXXX.html can still be accessed when an
explicit URL is provided (that's what crawlers already have).

Signed-off-by: black-dragon74 <niryadav@redhat.com>

### CODE REVIEW

**Comment by black-dragon74** [comment]
CC for review: @kshithijiyer @aravindavk @Shwetha-Acharya 

Regards

**Comment by aravindavk** [comment]
Changes looks good, but I don't see any issues with page link publicly visible. Google verifies that link is available from respective domain. This is to verify domain ownership.

**Comment by black-dragon74** [comment]
> Changes looks good, but I don't see any issues with page link publicly visible

Hi Aravinda, Thanks for review.

The link would still be publicly accessible. It's just that it is removed from the Navigation Menu because it is not related to our docs, right? Clicking on this link leads to an ugly HTML page (without nav), which in turn is bad UX. Google Search Console requires this HTML to be present on the server to verify the domain ownership, having a defined permalink isn't necessary.

To sum it up, everything works the same, just the menu that someone would click on to get to this page is not there.

Regards
